.changeset/config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@1.6.0/schema.json",
+  "changelog": "@changesets/cli/changelog",
+  "commit": false,
+  "linked": [],
+  "access": "restricted",
+  "baseBranch": "master",
```

Comment:
not `"develop"`?

Reply:
Ah, good question, this was the default output from the init. But this should defs be changed :)